x-pack/plugins/observability/public/hooks/use_fetch_rule_types.ts

```diff
@@ -65,13 +65,11 @@ export function useFetchRuleTypes({
       refetchOnWindowFocus: false,
       onError: (error: Error) => {
         toasts.addError(error, {
-          title: i18n.translate('xpack.observability.ruleDetails.ruleLoadError', {
-            defaultMessage: 'Unable to load rule. Reason: {message}',
-            values: {
-              message:
-                error instanceof Error ? error.message : typeof error === 'string' ? error : '',
-            },
+          title: i18n.translate('xpack.observability.ruleDetails.ruleTypeLoadError', {
+            defaultMessage: 'Unable to load rule type.',
```

Comment:
Remove the message from the title, as it's duplication and appears in the toast body.
Fix the key name, this one is for fetching the rule type, and it uses the same key for fetching the rule.